.size-snapshot.json

```diff
@@ -1,25 +1,25 @@
 {
   "dist/react-beautiful-dnd.js": {
-    "bundled": 357308,
-    "minified": 138768,
-    "gzipped": 40712
+    "bundled": 393322,
+    "minified": 147989,
+    "gzipped": 41340
   },
   "dist/react-beautiful-dnd.min.js": {
-    "bundled": 303784,
-    "minified": 113686,
-    "gzipped": 32910
+    "bundled": 327193,
+    "minified": 117669,
+    "gzipped": 33593
   },
   "dist/react-beautiful-dnd.esm.js": {
-    "bundled": 237952,
-    "minified": 125718,
-    "gzipped": 31356,
+    "bundled": 239473,
+    "minified": 123943,
+    "gzipped": 31513,
     "treeshaked": {
       "rollup": {
-        "code": 85891,
-        "import_statements": 832
+        "code": 30284,
```

Comment:
This is quite a nice drop. @TrySound I am not sure what this property is referring too. Can you please explain this property?

Reply:
This is the code in user bundle when you import nothing. Useful for ui libs like material ui. Does not mean a lot for this lib though.